# Results

## Target Selection 

As our proof-of-concept is to optimize bridge recombinase activity for AATD treatment, we identified three candidate target sequences (target 1-3) within the SERPINA1 locus. The targets were selected to 

(i) have high similarity to the IS621 wild-type target sequence

(ii) minimal similarity to any sequence in the E. coli genome to reduce off targets

(iii) be located between exon 1 and 2 as nearly all disease causing mutations occur downstream of exon 1. 

To assess the baseline recombination efficiency between the wild-type donor and the selected target sequences, we co-expressed two plasmids in the E. coli Marionette-Clo strain: one encoding the IS621 recombinase and another, the ”screening plasmid”, carrying the respective bRNA under the control of the PTet promoter along with two fluorescent reporter genes oriented in opposite directions (Figure 3A). 
The screening plasmid expresses mGreenLantern under the control of a constitutively active promoter; after inversion the two fluorescent reporter gene cassettes are flipped and mScarlet3 is placed downstream the promoter and expressed. The recombination efficiency for each target site is measured by flow cytometry and calculated as the percentage of mScarlet3-positive cells. Inversion is induced by addition of aTc which triggers bRNA A expression. 
After 15 h of induction targets 1-3 showed 6 % to 16 % of mScarlet3-
positive cells, whereas, the non-targeted target-sequence showed < 1 % positive cells (Figure 3B), confirming
target-specific inversion events. Similarly, we observed <1 % positive cells in the uninduced control in all
conditions, except target 2 and 3, where we observed 2% positive cells at the 15 h time point. This is indicative
of effective promoter repression in the absence of the inducer. Furthermore, we were only able to observe
a change in fluorescent protein expression after 15 hours of induction, as there were <1 % mScarlet positive
cells 2 hours after induction across all conditions. Lastly, flipping events were confirmed by Oxford Nanopore
sequencing. Target 1 showed the highest activity and was selected as the primary target sequence from this
point forward.

### Insert figure 


## Validation of the selection logic
